examples/lorem-ipsum/README.md

```diff
@@ -22,6 +22,6 @@ The `example-codebases` directory contains example projects connected to your Di
 
 ## Quickstart
 
-To quickly experience how Diez works, simply run `yarn demo` from `design-language`. This command will use Diez to compile a JavaScript SDK for your design language and link it to the example web project located in `codebases/web`, then start the example web project in your browser.
+To quickly experience how Diez works, simply run `yarn/npm demo` from `design-language`. This command will use Diez to compile a JavaScript SDK for your design language and link it to the example web project located in `codebases/web`, then start the example web project in your browser.
```

Comment:
Probably worth splitting these out, as long as we're using backticks for literal snippets.  "`yarn demo` or `npm demo`"